app/models/vim_performance_tag_value.rb

```diff
@@ -127,7 +127,7 @@ def self.build_for_association(parent_perf, assoc, options = {})
   def self.get_metrics(resources, timestamp, capture_interval_name, vim_performance_daily, category)
     if vim_performance_daily
       MetricRollup.with_interval_and_time_range("hourly", (timestamp)..(timestamp+1.day)).where(:resource => resources)
-          .for_tag_names(category, "") # append trailing slash
+          .for_tag_names([category, ""]) # append trailing slash
```

Comment:
if you look at the comment, this is requesting `"%category/%"` not requesting `[category,""]`

I do find these searches confusing with the multi levels

Reply:
@kbrock I added a test for this change. Can you merge if you're ok with it? Libor had approved but he no longer has merge rights